ci-operator/step-registry/fips-check/image-scan/fips-check-image-scan-ref.metadata.json

```diff
@@ -0,0 +1,10 @@
+{
+	"path": "fips-check/image-scan/fips-check-image-scan-ref.yaml",
+	"owners": {
+		"approvers": [
+			"mrunalp",
+			"swghosh",
+			"xingxingxia"
```

Comment:
Please add @rphillips here.

Reply:
I believe this is autogenerated from the OWNERS file. It probably needs to be added there, then regenerated.

Reply:
Added.